saleor/app/app_manifest_sample.json

```diff
@@ -1,14 +1,14 @@
 {
-  "id": "saleor.app.sample",
-  "version": "1.0.0",
-  "name": "My Wonderful App",
-  "about": "My Wonderful App is a wonderful App for Saleor.",
-  "permissions": ["MANAGE_USERS", "MANAGE_STAFF"],
-  "appUrl": "http://localhost:3000/app",
-  "configurationUrl": "htpp://localhost:3000/configuration",
-  "tokenTargetUrl": "http://localhost:3000/register",
-  "dataPrivacy": "Lorem ipsum",
-  "dataPrivacyUrl": "http://localhost:3000/app-data-privacy",
-  "homepageUrl": "http://localhost:3000/homepage",
-  "supportUrl": "http://localhost:3000/support"
+    "id": "saleor.app.sample",
+    "version": "1.0.0",
+    "name": "My Wonderful App",
+    "about": "My Wonderful App is a wonderful App for Saleor.",
+    "permissions": ["MANAGE_USERS", "MANAGE_STAFF"],
+    "appUrl": "http://localhost:3000/app",
+    "configurationUrl": "htpp://localhost:3000/configuration",
+    "tokenTargetUrl": "http://localhost:3000/register",
+    "dataPrivacy": "Lorem ipsum",
+    "dataPrivacyUrl": "http://localhost:3000/app-data-privacy",
+    "homepageOnMarketUrl": "",
```

Comment:
Why such a weird name?

Reply:
So what do you propose?

Reply:
This should be a URL which points on App's page on the marketplace

Reply:
What's wrong with `homepageUrl`?

Reply:
It doesn't emphasize that we expect the marketplace's URL.  We need to have a URL which we will pin to button `Visist on marketplace`